docs/intro/tutorial02.md

```diff
@@ -0,0 +1,116 @@
+# Writing your first app, part 2
+
+This tutorial begins where [Tutorial 1](tutorial01.md) left off. We’ll walk through the process to set up your personal
+registry and publish your application to that registry.
+
+For this tutorial, we will be creating a registry hosted on Microsoft Azure, but there are other cloud providers that
+provide their own solutions, and you can [run one on your own infrastructure](https://github.com/docker/distribution),
+too!
+
+## What is a registry, and what is wasm-to-oci?
+
+A registry allows you to store your local WebAssembly modules in the cloud. With a registry, you can backup your
+personal modules, share your projects, and collaborate with others.
+
+wasm-to-oci is an open source project that understands how to communicate with a registry. It takes a module you've
+built locally on your computer and publishes it to the registry, making it publicly available for others to access.
+
+## Create a registry
```

Comment:
I would also add a note about using upstream Docker Distribution.

Reply:
I made a note about that in part 1 under "prerequisites", as well as under the second paragraph in this section as preamble. I wanted to make it SUPER clear that this tutorial goes through Azure to focus on the important bits: building an app, pushing to a registry and running it with Krustlet, but if the user wants to use Docker Distribution to run on their own infra, they can do that if they desire.

Should I move that further down, or were you looking for a name drop here? :)

Reply:
I think the explanation from the beginning makes more sense here, but feel free to disagree, I don't have such a strong opinion here.

Reply:
I'll add it here too for good measure. :)